Sources/SwiftSyntaxBuilder/BuildablesConvenienceInitializers.swift.gyb

```diff
@@ -23,36 +23,36 @@ import SwiftSyntax
 
 % for node in SYNTAX_NODES:
 %   if node.is_buildable():
-%     has_syntax_collection_child = False
+%     should_create_convenience_initializer = False
 %     for child in node.children:
 %       child_node = NODE_MAP.get(child.syntax_kind)
-%       if child_node and child_node.is_syntax_collection():
-%         has_syntax_collection_child = True
+%       child_token = SYNTAX_TOKEN_MAP.get(child.syntax_kind)
+%       if (child_node and child_node.is_syntax_collection()) or (child_token and not child_token.text) or (child.syntax_kind is "IdentifierToken"):
```

Comment:
Shouldn’t `child.syntax_kind is "IdentifierToken"` already be covered by `child_token and not child_token.text`. `IdentifierToken`s should always produce a `child_token` and don’t have  `text` set. Or am I missing something here? (same applies a bit further down)